packages/flame/lib/src/components/spawn_component.dart

```diff
@@ -107,6 +109,9 @@ class SpawnComponent extends Component {
   /// Whether the timer automatically starts or not.
   final bool autoStart;
 
+  /// Whether the timer should start when the [SpawnComponent] is loaded.
+  final bool spawnWhenLoaded;
```

Comment:
`spawnWhenMounted` I would say is more accurate?

Reply:
But this happens on the onLoad phase doesn't it? That is when the `TimerComponent` is being created. Also this name would be consistent with `tickWhenLoaded` from the TimerComponent.

Reply:
Hmm, true, it should definitely be consistent. I think in reality it will happen when it is mounted though, because `onLoad` of `SpawnComponent` won't wait until the `TimerComponent` is mounted. But keep it like it is, it matters very little to the user.